gno.land/pkg/gnoweb/components/ui/toc_source.html

```diff
@@ -0,0 +1,67 @@
+{{- define "ui/toc_source" }}
+<div class="space-y-4">
+  <!-- Regular Files Section -->
+  <div>
+    <ul class="list-none space-y-2">
+      {{ range .RegularFiles }}
+      <li>
+        <a class="flex items-center gap-2 line-clamp-2 hover:text-green-600 hover:underline" href="{{ .Link }}">
+          <svg class="w-4 h-4 shrink-0">
+            <use href="#ico-{{ $.Icon }}"></use>
+          </svg>
+          {{ .Text }}
+        </a>
+      </li>
+      {{ end }}
+    </ul>
+  </div>
+
+  <!-- Test Files Section -->
+  {{ if .TestFiles }}
+  <details class="group">
+    <summary class="flex items-center cursor-pointer">
+      <h3 class="font-medium text-sm">Test Files</h3>
+      <svg class="w-4 h-4 ml-1 transform group-open:rotate-180 transition-transform my-auto" fill="none" stroke="currentColor" viewBox="0 0 24 24">
+        <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M19 9l-7 7-7-7"></path>
+      </svg>
+    </summary>
+    <ul class="list-none space-y-2 pl-2 mt-2">
```

Comment:
```suggestion
    <ul class="list-none space-y-2 pl-5 mt-2">
```
For a better alignement